perllib/FixMyStreet/App/Controller/Report.pm

```diff
@@ -454,6 +454,23 @@ sub inspect : Private {
                 $redirect_uri = $c->uri_for_action('my/planned');
             }
 
+            # Sign up to alerts by default
+            my $options = {
```

Comment:
I'm thinking User would be best, as it's acting on the user, rather than the problem